<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackedWidget</class>
 <widget class="QStackedWidget" name="StackedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="page0">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>176</width>
      <height>44</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="Nextbtn">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Backbtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>651</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Native卡</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Java卡</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>请选择卡类型</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>请选择运营商</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <item>
        <property name="text">
         <string>中国移动</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中国联通</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中国电信</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>请输入项目名称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="ProjectName"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>请输入测试人员</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="StuffName"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>请输入芯片型号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="ChipName"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="page1">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>176</width>
      <height>44</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="Nextbtn_2">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Backbtn_2">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>文件名称及说明</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>701</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>文档命名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>通过</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N/A</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QTextEdit" name="ScriptName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>文档说明</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_4">
       <item>
        <property name="text">
         <string>通过</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N/A</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="ScriptDescribe">
       <property name="placeholderText">
        <string>包括产品需要预置的功能及修改说明</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>文档特殊说明</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_5">
       <item>
        <property name="text">
         <string>通过</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N/A</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QTextEdit" name="SpecialDescribe">
       <property name="placeholderText">
        <string>见: 文档特别注意事项.doc</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>文档禅道</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_6">
       <item>
        <property name="text">
         <string>通过</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N/A</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTextEdit" name="ScrpitPath">
       <property name="placeholderText">
        <string>禅道任务ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="page">
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>176</width>
      <height>44</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <widget class="QPushButton" name="Nextbtn_3">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Backbtn_3">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
